configs/nextflow.config

```diff
@@ -262,6 +279,22 @@ process {
     withLabel:time_queue_from_week {
         time = { check_max( escalate_queue_time( 'week', task ), 'time' ) }
     }
+
+    withLabel:time_queue_from_small_slow2 {
```

Comment:
I'd like to keep 2 in the label so that if we want to make it increment even more slowly we can. at that rate we could make a function that's generic though. should that be the motivation for the next PR that makes cleaner names?